go.mod

```diff
@@ -69,7 +69,8 @@ require (
 	golang.org/x/net v0.0.0-20191002035440-2ec189313ef0
 	golang.org/x/sys v0.0.0-20200202164722-d101bd2416d5
 	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2
-	google.golang.org/grpc v1.24.0
+	google.golang.org/grpc v1.27.0
```

Comment:
We intentionally stick to specific proto versions (see [readme](https://github.com/lightningnetwork/lnd/blob/master/lnrpc/README.md#generate-protobuf-definitions)), so we don't need to bump any versions here.